src/components/SocialBtns/SocialBtns.js

```diff
@@ -1,30 +1,50 @@
-import React, { PropTypes } from 'react';
+/* global FB, location */
+import React, { Component, PropTypes } from 'react';
 import FaFacebookSquare from 'react-icons/lib/fa/facebook-square';
 import FaTwitter from 'react-icons/lib/fa/twitter';
 import styles from './SocialBtns.sass';
+import fbSdk from './fb_sdk';
 
-const SocialBtns = ({
-  fbLink = 'https://www.facebook.com',
-  twitterLink = 'https://twitter.com',
-  iconSize = '25'
-}) => {
-  return (
-    <div className={styles.socialBtns}>
-      <a
-        href={fbLink}
-        className={styles.facebookBtn}
-      >
-        <FaFacebookSquare size={iconSize} />
-        <span>Share on Facebook</span
-      >
-      </a>
-      <a href={twitterLink} className={styles.twitterBtn}>
-        <FaTwitter size={iconSize} />
-        <span>Share on Twitter</span>
-      </a>
-    </div>
-  );
-};
+class SocialBtns extends Component {
+
```

Comment:
Whenever you add a class, if you were to be binding functions in the class, you would add as the first method in the class:
```
constructor(props){
     super(props);
}
```
See https://facebook.github.io/react/docs/react-component.html#constructor for more details

Otherwise, your syntax looks good!